### DISCUSSION TITLE
Is posible to move the pivot from a specific part of the voxel?

### DISCUSSION THREAD

**Discussion by CortesJoan** [vengi-voxel/vengi#439] [Q&A]
I am planning to move up from MagicaVoxel to another voxel tool. But I need to know if I can do this first with this one. Is it possible to move the pivot of a specific part of the voxel, like I have a human and want to move the pivot of the leg to avoid that the leg rotates around the center with Magical Voxel I can't so that's why I'm asking if the Vengi engine can.

**Comment by mgerhardy** [comment]
Yes, that is possible

![Bildschirmfoto vom 2024-04-02 17-04-31](https://github.com/vengi-voxel/vengi/assets/577713/ea3629de-0193-40c8-9523-4f3aa669d59e)
![Bildschirmfoto vom 2024-04-02 17-04-05](https://github.com/vengi-voxel/vengi/assets/577713/2268bbdd-034a-4cbe-8534-219e1e26ed84)

But there are a few open issues regarding pivots

* https://github.com/vengi-voxel/vengi/issues/337
* https://github.com/vengi-voxel/vengi/issues/426

**Reply by mgerhardy** [reply]
But before switching from a stable software like MagicaVoxel, keep in mind that voxedit is still in beta and heavy under development. I wouldn't suggest switching to it for any reasonable work right now. I would still be grateful for bug reports and feedback from a real voxel artist though ;)
